File: 9.py
target = int ( input ("Triplet Test: "))

for a in range(1, round(target / 3)):
    #Taking advantage of a < b < c instead of 1,1000
    for b in range(a, round(target / 2)):
        #Taking advantage of a < b < c instead of 1,1000
        c = target - a - b
        triplet = a * a + b * b == c * c
        
        if triplet:
            print(f'Triplet: {a} {b} {c}\n' +
                  f'Sum: {a} + {b} + {c} = {a+b+c}\n' +
                  f'Product: {a} × {b} × {c} = {a*b*c}')

File: 9.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Euler_Project_9
{
    class Program
    {
        static void Main(string[] args)
        {
            const int target = 1000;

            for (int a = 0; a < target / 3; a++) {
                for (int b = a; b < target / 2; b++) {
                    int c = target - a - b;
                    bool triplet = a * a + b * b == c * c;

                    if (triplet) {
                        Console.WriteLine($"Triplet: {a}, {b}, {c}\n" +
                                          $"Sum: {a} + {b} + {c} = {a+b+c}\n" +
                                          $"Product: {a} × {b} × {c} = {a*b*c}");
                    }
                }
            }
            Console.ReadKey();
        }
    }
}
